### PR TITLE
Dcsobral/ch8673/parallelize workload on dynamodb etl script

### PR DESCRIPTION
**Pull request by dcsobral**

### CODE REVIEW

File: apps/dynamodb-etl/src/main.rs [hunk 1]
```diff
@@ -44,11 +42,11 @@ const DEFAULT_TEXT_PATH: &str = ".projectData.S";
 #[structopt(name = "dynamodb-etl", about = "", author = "")]
 struct Opt {
     /// Binary data path
-    #[structopt(short, long, default_value = ".projectBinaryData.B")]
+    #[structopt(short, long, raw(default_value = "DEFAULT_BIN_PATH"))]
```
Comment (dcsobral) [inline apps/dynamodb-etl/src/main.rs:45]: These are unrelated changes. I got a reply about how to do this, and tried it out. In the end, I committed here because I was tired of stashing it, and I was worried about _forgetting_ about it if I didn't unstash it.

**Comment by dcsobral** [comment]
I forgot one change, so there's one more commit incoming.

**Comment by dcsobral** [comment]
Done.

**Review by nicflores — APPROVED** [review]
